src/CNCOIL.CPP

```diff
@@ -2011,7 +2011,7 @@ x	}
 				we = ahcc.wsatMinTEvap;
 				//ahcc.capt = (ahcc.hsatMinTEvap - hena) * menFOn * ahcc.efecO;		// reduced capacity
 			}
-#ifdef DEBUG
+#if defined( _DEBUG)
```

Comment:
Are there other changes like this that need to be made throughout the code?

Reply:
Yes, all DEBUGs should be reviewed to see if they might be _DEBUG

Reply:
What is the difference between:

1. `DEBUG` and `_DEBUG`?
2. `#ifdef` and `#if defined()`?

Reply:
I think (?) `_DEBUG` allows for conditionally-compiled code, whereas `DEBUG` is a directive that compiles in the debug symbols and also modifies name-mangling (I've found I have to undef DEBUG if I want to export a C-DLL without name mangling)